docs/UserGuide.md

```diff
@@ -184,6 +201,21 @@ Example of usage:
 
 ***
 
+### Resetting a budget: `RES`
+Resets te budget to be 0.00 for the user.
+
+Format: `RES`
+
+Example of usage: 
+
+`RES`
```

Comment:
I think for the simple commands we aren't including an example of usage, since its the same as in the format

Reply:
Okiesz, will make the changes! Thank you :)